modules/qt/hm.nix

```diff
@@ -0,0 +1,97 @@
+{
+  pkgs,
+  config,
+  lib,
+  ...
+}:
+{
+  options.stylix.targets.qt = {
+    enable = config.lib.stylix.mkEnableTarget "QT" pkgs.stdenv.hostPlatform.isLinux;
+    platform = lib.mkOption {
+      description = ''
+        Platform for QT.
+
+        Defaults to the standard platform of the configured DE in NixOS when
+        `stylix.homeManagerIntegration.followSystem = true`.
+
+        Fallback to qtct.
```

Comment:
This might be redundant:

```suggestion
```

Reply:
This is for standalone home-manager users who don't get the correct setting from NixOS.

Reply:
I did remove it from the nixos module though.

Reply:
I mean, is this information not redundant because it essentially just states the `default` value inside the `description`?